crates/bevy_render/src/render_graph/node_slot.rs

```diff
@@ -51,7 +51,12 @@ impl ResourceSlots {
 
     pub fn get(&self, label: impl Into<SlotLabel>) -> Option<RenderResourceId> {
         let slot = self.get_slot(label).unwrap();
-        slot.resource.clone()
+        slot.resource
+    }
+
+    pub fn get_mut(&mut self, label: impl Into<SlotLabel>) -> &mut Option<RenderResourceId> {
```

Comment:
Same here with changing this to a `Option<&mut RenderResourceId>`

Reply:
Same as above.

Reply:
I removed this in favor of using set.